debezium-core/src/test/java/io/debezium/config/ConfigurationTest.java

```diff
@@ -145,15 +145,6 @@ public void defaultDdlFilterShouldFilterOutFlushRelayLogs() {
         assertThat(ddlFilter.test("FLUSH RELAY LOGS")).isTrue();
     }
 
-    @Test
```

Comment:
@pushpavanthar Thanks for the contribution! Would it be possible to add an integration test that would verify this functionality? It seems to me that adding few tables to `replicate-ignore-table` mysql options might do the trick to reproduce the necessary setup.

Reply:
Hi @jpechane, Since this functionality involves Slave MySQL configuration changes, I'm stuck in writing Integration Test case. Could you please help me on applying replication filters in the container MySQL instance? I tried to execute below commands from jdbc connection in my test case.  However, it failed to updated the Slave filters.
`STOP SLAVE SQL_THREAD;
CHANGE REPLICATION FILTER REPLICATE_IGNORE_TABLE = (db1.table1, db2.table1);
START SLAVE SQL_THREAD;`